src/ethereum_test_forks/forks/forks.py

```diff
@@ -1000,22 +1000,17 @@ def solc_min_version(cls) -> Version:
         return Version.parse("1.0.0")  # set a high version; currently unknown
 
 
-class PragueEIP7692(  # noqa: SC200
-    Prague,
-    transition_tool_name="Prague",  # Besu enables EOF at Prague
-    blockchain_test_network_name="Prague",  # Besu enables EOF at Prague
-    solc_name="cancun",
-):
+class Osaka(Prague, solc_name="cancun"):
```

Comment:
Should this be added to the __all__ section in `__init__.py` for this package?

Reply:
Indeed, thanks!